sdk/eventgrid/eventgrid/README.md

```diff
@@ -252,45 +252,34 @@ await client.send([
 
 ### Deserializing an Event
 
-`EventGridDeserializer` can be used to deserialize events delivered by Event Grid. When deserializing an event, you need to know the schema used to deliver the event. In this example we have events being delivered to an Azure Service Bus Queue in the Cloud Events schema. Using the Service Bus SDK we can receive these events from the Service Bus Queue and then deserialize them using `EventGridDeserializer` and use `isSystemEvent` to detect what type of events they are.
+`EventGridDeserializer` can be used to deserialize events delivered by Event Grid. In this example we have a cloud event that is deserialized using `EventGridDeserializer` and use `isSystemEvent` to detect what type of events they are.
 
 ```js
-const { ServiceBusClient } = require("@azure/service-bus");
-const { DefaultAzureCredential } = require("@azure/identity");
 const { EventGridDeserializer, isSystemEvent } = require("@azure/eventgrid");
 
-const client = new ServiceBusClient("<service bus hostname>", new DefaultAzureCredential());
-
-const receiver = client.createReceiver("<queue name>", "peekLock");
-
-const consumer = new EventGridDeserializer();
-
-async function processMessage(message) {
-  // When delivering to a Service Bus Queue or Topic, EventGrid delivers a single event per message.
-  // so we just pluck the first one.
-  const event = (await consumer.deserializeCloudEvents(message.body))[0];
+async function main() {
+  const deserializer = new EventGridDeserializer();
+  const message = {
+    id: "5bc888aa-c2f4-11ea-b3de-0242ac130004",
+    source:
+      "/subscriptions/<subscriptionid>/resourceGroups/dummy-rg/providers/Microsoft.EventGrid/topics/dummy-topic",
+    specversion: "1.0",
+    type: "Microsoft.ContainerRegistry.ImagePushed",
+    subject: "",
```

Comment:
nit: can we add a subject here? It looks a bit odd for a sample that this is required but passing empty string

Reply:
Done